backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -90,6 +91,7 @@
   private final SortingHandler sortingHandler;
   private final FeatureHandler featureHandler;
   private final CreateExperimentRunHandler createExperimentRunHandler;
+  private final Config config = Config.getInstance();
```

Comment:
This will be removed in other PR so while merging other this will be removed from here as well